hbase-server/src/main/java/org/apache/hadoop/hbase/master/MasterRpcServices.java

```diff
@@ -1696,24 +1695,42 @@ private void checkHFileFormatVersionForMob() throws IOException {
     }
   }
 
+  // This method implements Admin getRegionInfo. On RegionServer, it is
```

Comment:
Make this javadoc please?

Reply:
BIt odd. This implements and Interface that has two different implementations... one in RS and other in M. Seemed best as comment. Made it javadoc as you requested.